.vscode/launch.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
FYI, You can add this to your .gitignore file so that it's not added to the repo. https://www.freecodecamp.org/news/gitignore-what-is-it-and-how-to-add-to-repo/

Not mandatory or something you're expected to know, but thought you may be interested